src/test/java/pl/taskyers/taskybase/integration/IntegrationBase.java

```diff
@@ -44,17 +42,18 @@
     protected ProjectInvitationTokenRepository projectInvitationTokenRepository;
     
     @Autowired
-    protected StatusEntryRepository statusEntryRepository;
+    protected EntryEntityRepository entryEntityRepository;
     
     protected final ObjectMapper objectMapper = new ObjectMapper().configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
     
     protected final String DEFAULT_USERNAME = "u1";
     
-    protected CustomizableEntryDTO createDTO(String value, String textColor, String backgroundColor) {
+    protected CustomizableEntryDTO createDTO(String value, String textColor, String backgroundColor,String type) {
         CustomizableEntryDTO customizableEntryDTO = new CustomizableEntryDTO();
         customizableEntryDTO.setValue(value);
         customizableEntryDTO.setTextColor(textColor);
         customizableEntryDTO.setBackgroundColor(backgroundColor);
+        customizableEntryDTO.setEntryType(type);
```

Comment:
To chyba można przerzucić do tej klasy z testami, bo wtedy myślałem, że będzie używane w innych miejscach jeszcze